examples/astro/README.md

```diff
@@ -0,0 +1,40 @@
+# Welcome to [Astro](https://astro.build)
```

Comment:
Could this README follow the pattern of the other examples here? It can keep everything, just needs to mention deploying to Vercel and have the deploy button.